.github/workflows/codeql.yml

```
@@ -44,8 +44,13 @@ jobs:
           languages: ${{ matrix.language }}
 
       - name: Autobuild
+        if: matrix.language != 'cpp'
         uses: github/codeql-action/autobuild@v3
 
+      - name: Build dummy C file for CodeQL
+        if: matrix.language == 'cpp'
+        run: clang -c internal/cryptokit/cryptokit_codeql.c -o internal/cryptokit/cryptokit_codeql.o
```

Would this work? It does locally, but don't know if CodeQL will like it.

```suggestion
        run: clang -c internal/cryptokit/cryptokit.h -o /dev/null
```

---

it works!